packages/api/app/Models/Technology.js

```diff
@@ -85,20 +86,47 @@ class Technology extends Model {
 		query.with('terms.taxonomy');
 	}
 
+	static async scopePublished(query) {
+		query.where({ status: 'published' });
+	}
+
 	/**
 	 * Query scope to get the technology either by id or slug
 	 *
 	 * @param {object} query The query object.
 	 * @param {number|string} technology The technology id or slug
 	 *
-	 * @returns {object}
 	 */
 	static scopeGetTechnology(query, technology) {
 		if (Number.isInteger(Number(technology))) {
-			return query.where({ id: technology });
+			query.where({ id: technology });
+		} else {
+			query.where({ slug: technology });
+		}
+	}
+
+	/**
+	 * Checks if user can access unplisheds technologies
+	 *
+	 * @param {number|string} technology The technology id or slug
+	 * @param {object} user Auth User
+	 * @returns {object}
+	 */
+	static async canAccessUnPublishedTechnology(technology, user) {
+		const userRole = await user.getRole();
+		if (userRole === roles.ADMIN) return true;
+
+		const isRelatedUser = await technology.checkResponsible(user);
+		if (isRelatedUser) return true;
+
+		const technologyReviewer = await technology.getReviewer();
+
+		let isReviewer = false;
+		if (technologyReviewer) {
+			if (technologyReviewer.user_id === user.id) isReviewer = true;
 		}
 
-		return query.where({ slug: technology });
+		return isReviewer;
```

Comment:
```suggestion
		return technologyReviewer && technologyReviewer.user_id === user.id;
```

com essa mudança vc não precisa definir os isReviewer